pkg/stats/stats.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package stats
+
+import (
+	"fmt"
+
+	"sigs.k8s.io/cli-utils/pkg/apply/event"
+)
+
+type Collector struct {
+	Stats Stats
+}
+
+func NewCollector() *Collector {
+	return &Collector{}
+}
+
+func (n *Collector) Run(eventChannel <-chan event.Event) <-chan event.Event {
```

Comment:
How about "Collect" as the func name?

Reply:
Yeah, that is a better name. I have updated the PR.